app/src/main/java/com/infomaniak/mail/data/api/ApiRepository.kt

```diff
@@ -290,6 +291,14 @@ object ApiRepository : ApiRepositoryCore() {
         return callApi(ApiRoutes.ai(), POST, body, HttpClient.okHttpClientLongTimeout)
     }
 
+    private fun checkFeatureFlag(featureFlagType: FeatureFlagType): ApiResponse<Boolean> {
+        return callApi(ApiRoutes.featureFlag(featureFlagType.apiName), GET)
+    }
+
+    fun checkAiFeatureFlag(): ApiResponse<Boolean> {
```

Comment:
Why have you created this function ? 
It would be already straightforward to use checkFeatureFlag in code

Reply:
I agree it probably enough to call `checkFeatureFlag(FeatureFlagType.AI)` actually